app/gateway/traffic-control/proxy.md

```diff
@@ -0,0 +1,191 @@
+
+1. {{site.base_gateway}} listens for HTTP traffic on its configured
+proxy port(s) (`8000` and `8443` by default) and L4 traffic on explicitly configured
+`stream_listen` ports.
+1. {{site.base_gateway}} will evaluate any incoming HTTP request or L4 connection against the Routes you have configured and try to find a matching one.
+1. If multiple Routes match, the {{site.base_gateway}} router then orders all defined Routes by their priority and uses the highest priority matching Route to handle a request.
+1. If a given request matches the rules of a specific route, {{site.base_gateway}} will
+run any global, Route, or Service [plugins]() before it proxies the request. They are run in the following order: Route, Service <!--when does the global config get run?--> These configured plugins will run their `access` phase, which you can find more
+information about in the [Plugin development guide][plugin-development-guide].
+1. {{site.base_gateway}} implements [load balancing]() capabilities to distribute proxied
+requests across a pool of instances of an upstream service.
+1. Once {{site.base_gateway}} has executed all the necessary logic (including plugins), it is ready to forward the request to your upstream service. This is done via Nginx's [`ngx_http_proxy_module`][ngx-http-proxy-module].
+1. {{site.base_gateway}} receives the response from the upstream service and sends it back to the
+downstream client in a streaming fashion. At this point, {{site.base_gateway}} executes
+subsequent plugins added to the route and/or service that implement a hook in
+the `header_filter` phase.
+
+## Listeners
+
+From a high-level perspective, {{site.base_gateway}} listens for HTTP traffic on its configured
+proxy ports: `8000` and `8443` by default and L4 traffic on explicitly configured
+`stream_listen` ports. {{site.base_gateway}} will evaluate any incoming
+HTTP request or L4 connection against the routes you have configured and try to find a matching
+one.
+
+{{site.base_gateway}} exposes several interfaces which can be tweaked by the following configuration properties:
+
+- `proxy_listen`: Defines a list of addresses/ports on which {{site.base_gateway}} will
+  accept **public HTTP (gRPC, WebSocket, etc) traffic** from clients and proxy
+  it to your upstream services (`8000` by default).
+- `admin_listen`, which also defines a list of addresses and ports, but those
+  should be restricted to only be accessed by administrators, as they expose
+  Kong's configuration capabilities: the **Admin API** (`8001` by default).
+    {:.important}
+   > **Important**: If you need to expose the `admin_listen` port to the internet in a production environment,
+   > {% if_version lte:2.8.x %}[secure it with authentication](/gateway/{{include.release}}/admin-api/secure-admin-api/).{% endif_version %}{% if_version gte:3.0.x %}[secure it with authentication](/gateway/{{include.release}}/production/running-kong/secure-admin-api/).{% endif_version %}
```

Comment:
For reviewers: I couldn't find the Admin API in the dev site, does anyone have a link to it?